example/mii_core_data_set/generate_cds.py

```diff
@@ -789,6 +815,12 @@ def apply_additional_profile_rules(named_profile: Tuple[str, UIProfile]):
     container.stop()
     container.remove()
 
+    if args.add_delete_statements:
+        insert_content(base_file=f'{os.getcwd()}/R__Load_latest_ui_profile.sql',
+                       content_to_insert='resources/delete_statements.sql',
+                       insert_after='SET row_security = off;')
```

Comment:
@michael-82  what if insert_after has more than one match - can one then still guarantee is inserted in the right place?

Reply:
Absolutely not. It will just insert it after the first occurrence of the search string.